### ISSUE TITLE
Evaluation Plots for LambdaRankNN

### ISSUE THREAD

**Issue by sureshgorakala** [liyinxiao/LambdaRankNN#5]
Hi @liyinxiao,

Firstly great work, made my life easy to use the Learning2Rank models using Keras.
Few questions I got while using the library:

1. what is the loss function that is being used during training process - is it binary_cross_entropy? 
2. If yes, to above question, How do I plot the evaluation graphs for training & evaluation sets in terms of ndcg 
2. how do I extend the eval metrics to MAP ?

Any guidance or support is appreciated?

Thanks in advance

**Comment by liyinxiao** [comment]
Thanks @sureshgorakala , and sorry for taking so long to respond:
1. Yes! binary_crossentropy is the loss function.
2. You can make a plot here when evaluate() is called https://github.com/liyinxiao/LambdaRankNN/blob/master/LambdaRankNN/__init__.py#L162-L165
3. You can code up a MAP helper function for that purpose here https://github.com/liyinxiao/LambdaRankNN/blob/master/LambdaRankNN/__init__.py#L161
